### PR TITLE
feat: add Lisk chain testnet

### PR DESCRIPTION
**Pull request by 0xtiti**
closes SUP-117

### CODE REVIEW

**Comment by 0xtiti** [comment]
Tx data:

L1 hash: 0x9df12dfabe1aa8605923b7740108aa6f49659aab730ba0054a77c9d1cbb5c983

Screen Shot 2024-05-10 at 16 30 10

Screen Shot 2024-05-10 at 16 29 41

error: RPC error

File: src/data/chains/lisk.ts [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+import { OpContracts } from '~/types';
+import { l2Contracts } from '~/utils';
+
+// docs: https://documentation.lisk.com/contracts
+
+export const liskChainData = {
+  [liskSepolia.id]: {
+    logo: liskLogo.src,
+    apiUrl: 'https://rpc.sepolia-api.lisk.com/',
```
Comment (0xArdy) [inline src/data/chains/lisk.ts:13]: this should be the explorer API endpoint to fetch the contract ABIs in the custom transaction section, not the rpc endpoint

File: src/data/chains/redstone.ts [hunk 2]
```diff
@@ -51,7 +51,7 @@ export const redstoneExtraTokens = [
     name: 'Ether',
     symbol: 'ETH',
     decimals: 18,
-    logoURI: 'https://ethereum-optimism.github.io/data/frxETH/logo.svg',
+    logoURI: 'https://ethereum-optimism.github.io/data/ETH/logo.svg',
```
Comment (0xArdy) [inline src/data/chains/redstone.ts:54]: nice catch ty!

File: src/data/chains/lisk.ts (hunk 1 above)
Comment (0xtiti) [reply src/data/chains/lisk.ts]: I removed the apiUrl then. https://sepolia-blockscout.lisk.com/ this is the explorer and API section provides: only RPC api from blockscout https://docs.blockscout.com/for-users/api/rpc-endpoints which I dont think its the one we are looking for.

**Review by 0xArdy — APPROVED** [review]
lgtm!

**Review by 0xShaito — APPROVED** [review]

**Pull request merged**